FirebaseAuth/Tests/Unit/FIRSignInWithGameCenterTests.m

```diff
@@ -16,7 +16,13 @@
 
 #import <XCTest/XCTest.h>
 
-#import "FirebaseAuth/Sources/Public/FirebaseAuth/FIRAuthBackend.h"
+#if SWIFT_PACKAGE
+#import <FirebaseAuth-Swift.h>
+#else
+#import <FirebaseAuth/FirebaseAuth-Swift.h>
+#endif
```

Comment:
@paulb777, we chatted about this a few weeks ago, but here is an example of a difference between how CocoaPods and SPM distributes the generated Swift header.

- CocoaPods requires the `FirebaseAuth` subdirectory in the public headers directory so it places in there
- SPM mixed language support has been designed to avoid that requirement and it places it at the root of the public headers directory.

I also could just simplify this and do an `@import FirebaseAuth` but chose the IWYU route to emphasize that these tests increasingly depend on converted Swift types!